core/templates/pages/blog-dashboard-page/navbar/navbar-breadcrumb/blog-dashboard-navbar-breadcrumb.component.html

```diff
@@ -59,11 +59,14 @@
     color: #fff;
     font-family: 'Roboto', Arial, sans-serif;
     font-size: 1.2rem;
+    position: fixed;
+    top: 28px;
   }
 
   @media(max-width: 768px) {
     .oppia-blog-post-title {
       font-size: 1rem;
+      top: 35px
```

Comment:
Instead of setting a custom top for different screen sizes, shouldn't we just set align items to center or something similar?

Reply:
I tried text-align: center and align-tiems: center and it didn't work.

Reply:
Ah it looks like that was not the issue in the first place. The 'blog-title' div seems to have an incorrect `bottom: 4rem`, the correct approach here would likely be to remove that styling, please check.

<img width="315" alt="image" src="https://github.com/oppia/oppia/assets/73329256/fcfa2e98-9bb0-4443-9f68-4a217440a771">

Reply:
Yes, you are right. I removed the bottom property from both English and Arab lang and now it is centerized.